CHANGES.rst

```diff
@@ -2,12 +2,19 @@ Changelog
 =========
 
 
-1.0.1 (unreleased)
+1.1.1 (unreleased)
 ------------------
 
 - Nothing changed yet.
 
 
+1.1.0 (2023-02-14)
+------------------
+
```

Comment:
Depending on the tooling you use to create a new release, this is done automatically, `zest.releaser` does that for example.

I see on [this add-on's pypi's page](https://pypi.org/project/collective.splitsitemap/#history) that this release has not been made and only @frapell can actually publish new releases

Reply:
@gforcada is your pypi user `gforcada` as well? I can give you access if you want

Reply:
no, is `gilforcada` but I don't have much interest in this package, I reviewed the PR only 😓 you might give access to the `collective` user that exists on pypi to help manage releases of packages though 👍🏾

Reply:
done! :)